Stride.BepuPhysics/Processors/ContainerProcessor.cs

```diff
@@ -91,29 +92,29 @@ public override void Update(GameTime time)
                     bepuSim.RemainingUpdateTime -= bepuSim.SimulationFixedStep;
                     stepCount++;
                 }
-                //simStepWatch.Stop();
-
-                //parForWatch.Start();
 
+                #warning I don't think this should be user-controllable ? We don't provide control over the other parts of the engine when they run through the dispatcher and having it on or of doesn't (or rather shouldn't) actually change the result, just how fast it resolves
+                // I guess it could make sense when running on a low power device, but at that point might as well make the change to Dispatcher itself
```

Comment:
I think this was the big benefit of an easy toggle. I cant properly remember now but I thought there was an issue with mobile for Dispatcher calls or just multithread in general?

Although there is definitely a limit I think more control for little impact is a benefit, this could probably be simplified though.

Reply:
Eh, it's not that big of a deal to me, feel free to get rid of the warning

Reply:
I experienced significant performance improvements with Parallel.For vs for for big physique scenes. However, while simulating a scene involving low bodies count, Parallel was consuming excessive frame time unnecessarily. I haven't tested it since Doprez switched to Dispatcher.For, so we need to conduct a test to determine whether to continue using the switch bool or exclusively switch to it.

Reply:
There was a 2ms delay that was added by Parallel that seems to be gone with Dispatcher.